### DISCUSSION TITLE
2021-6-23-SecretDecks Meeting

### DISCUSSION THREAD

**Discussion by wlstylr** [SecretDecks/Documentation#54] [Meetings]
# Secret Decks Weekly Meeting
23rd June 2021

## Atendees
Dan, Tyler

#### ACO Structure
- Discussed Project Catalyst circle of delegates (I forgot the exact terminology).  
- Discussed how the golden share model for ACO might add value to both the ACO and Project Catalyst.  Could be a solid PC proposal for F6.  
- I guess funds would go directly into creating the legal entity, putting the funds in the reserve and allocating 90% of the reserve to the golden share that is being given to catalyst (Dan - Discord comment)
- So the 10% of funds received would go to creating the legal entity and the remaining 90% back to Catalyst in the form of the golden share.  The 90% would remain constant in value, and not correlate to additional value created through the ACO? (Tyler - Discord comment)
- GmbH costs 25,000 Euros.  (Dan)
- Dan volunteered to reach out to [Purpose](https://purpose-economy.org/en/) to explore and learn a bit more. 

#### Outreach / Community Building
- Should we start bringing more people into the community?
-  Need to have v. 0.01 voting ready (discord polls, typeform, airtable?) 
- Discord polling bot is probably easiest to implement
- Look into project catalyst voting app for long term solution.  Is it open source?
-  Need to create Github onboarding tutorials.  (Create an account, Leave an intro in discussions, Make a pull request) 
- Create Repo for tutorials

#### Content
- Article about the logo
-  Visual content for the medium articles Know Thy Self, and Meet Thy Self
-  Possibly The Magician card?

#### Artists
- Heather and Kat are working with Artists

#### To do
- Create Github onboarding tutorials - TW
- Polling bot for voting - DV

**Comment by DanM3rcurius** [comment]
@stephen-rowan We urgently need to meet to kickoff the governance design. In order to open up (wider than family & friends) to the public/wider community we need a basic off-chain voting (polling) mechanism in place. 
Tyler and me were thinking about Discord polling, as it would allow for lesser attack vectors since members have to register with mail and go through staged levels of "commitment" (so to say).
Until we have a proper on chain voting (polling) solution - eg. Catalyst voting  app but voting only with SD native token - this will have to suffice, i guess?!

It remains to be decided in the kickoff meeting: Who can vote on what? @SecretDecks/founding-core opinion is obviously welcome here!
🙏🏿 
D

**Reply by stephen-rowan** [reply]
It might be a good idea to reference the new Catalyst Circle voting system (see TH Slides [here](https://docs.google.com/presentation/d/1k_jnLp9Cm5jBsNan9VJoPjAm2Ko8jCSu-klwYA1HuDI/edit#slide=id.ge00bc5ea9a_0_2845)). So that we are aligned with Catalyst's first experiment in voting governance.

This is simply crowdsourced and then the result put in a spreadsheet (with a view to automate later). Starting with a human approach to voting is probably a good idea as it allows us to work out any wrinkles before things get locked in code ? 

## Here is the Catalyst Election Protocol made generic  ...

## Election on what is to be decided

Election space (voters go to space)

In room a facilitator will communicate voting options. 

Following that, a quick, synchronous voting process will take place.

Votes will be tallied immediately.

Results are captured.

## Election on what is to be decided (translated to SD Discord)

Election space (voters go to Discord voting channel)

In room a facilitator will communicate voting options (this can be done as a Discord post)

Following that, a quick, synchronous voting process will take place. (in our case an asynchronous window).

Votes will be tallied immediately (thumbs up/down)

Results are captured (as Discord reactions).

**Comment by DanM3rcurius** [comment]
Found this old task when going through meeting minutes
Contacted Purpose AG just now!
